docs/source/resources/scylladbclusters/basics.md

```diff
@@ -0,0 +1,218 @@
+# ScyllaDBClusters
+
+## Introduction
```

Comment:
since the ScyllaDBCluster (v1alpha1) is being introduced here and we're getting into the whole ScyllaCluster/ScyllaDBCluster (with no clear version indication in most places), I have a few general questions/suggestions:
- ScyllaDBCluster documentation could use more explicit messaging about it being in an alpha stage
- perhaps we should at least add a mention of ScyllaDBDatacenter somewhere, explaining that ScyllaCluster is meant to be replaced by it in the future?
- what's the overall plan for ScyllaDBDatacenter documentation?

Reply:
Added a note about it being technical preview. 
The place where both of these resources are introduced (overview) has a description pointing out the difference between those.
I don't think we should mention ScyllaDBDatacenter, currently it's internal resource, without clear path of it being pushed towards GA.

Reply:
lgtm for my part